### DISCUSSION TITLE
Other existing OSC xAPPs

### DISCUSSION THREAD

**Discussion by hullspokes** [srsran/oran-sc-ric#45] [Q&A]
First of all, thank you for creating the ORAN SC RIC in docker. It is so much simpler than the conventional deployment. 

Currently there are only examples for python xAPPs in the repo. What would I have to do to use the other xAPP frames provided by the SC (e.g. in Go, Rust, C++)? 

My second question is whether it is somehow possible to get other existing xAPPs (e.g. the traffic steering xAPP - https://github.com/o-ran-sc/ric-app-ts) to run with this test setting?

Thanks in advance and best regards!

**Comment by newt2105** [comment]
Hi @hullspokes, Did you find the solution? Can you create or custom some xapp?
